README.md

```diff
@@ -11,14 +11,15 @@ This package is developed by [valantic CEC Schweiz](https://www.valantic.com/en/
 ## Setup
 
 1. `composer require valantic/pimcore-elastica-bridge`
-1. Use the Pimcore UI
-    1. Open the Tools -> Bundles screen in Pimcore
-    1. Enable the bundle
-1. Use the Pimcore CLI
-    1. `console pimcore:bundle:enable ValanticElasticaBridgeBundle`
-1. If all else fails
-    1. Edit `var/config/extensions.php`
-    1. Add `"Valantic\\ElasticaBridgeBundle\\ValanticElasticaBridgeBundle" => TRUE,`
+1. Enable the bundle using one of the following methods
+    - Use the Pimcore UI
+        1. Open the Tools -> Bundles screen in Pimcore
+        1. Enable the bundle
+    - Use the Pimcore CLI
+        1. `bin/console pimcore:bundle:enable ValanticElasticaBridgeBundle`
+    - Edit files
+        1. Edit `var/config/extensions.php`
```

Comment:
Since we're already touching this, can you please take care of an extra bit? This method has been deprecated by Pimcore (https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/Upgrade_Notes/index.html#page_10-5-0, search for `var/config/extensions.php`). Can you please update this section accordingly? Thanks!

Reply:
Well, according to this release note we would also have to get rid of the first option. Sounds to me like adding it to `config/bundles.php` is the only option left? Does that already work now? Since this is also the Symfony standard way I would prefer that. However, Pimcore 11 is not yet release, right? Therefore I would stick with it as is for now, since there might even more stuff change.

Reply:
> Well, according to this release note we would also have to get rid of the first option.

Agreed.

> Sounds to me like adding it to config/bundles.php is the only option left?

The release notes didn't mention deprecating the commands so that option should still work.

> However, Pimcore 11 is not yet release, right? Therefore I would stick with it as is for now, since there might even more stuff change.

Works for me!